cob_hardware_config/robots/cob4-25/config/base_driver.yaml

```diff
@@ -0,0 +1,12 @@
+nodes:
+  fl_caster_rotation_joint:
```

Comment:
same values as in https://github.com/mojin-robotics/mojin_calibration_data/pull/44